rust/operator-binary/src/controller.rs

```diff
@@ -823,6 +823,7 @@ fn build_server_rolegroup_daemonset(
 
         match &user_info.backend {
             user_info_fetcher::Backend::None {} => {}
+            user_info_fetcher::Backend::Aas(_) => {}
```

Comment:
Does AAS not require authn?

Reply:
No it does not - that's also why it is still experimental. Or, it might support it? But at this point we do not support it, as it isn't clear how authn might be enabled if it is supported.

Reply:
Today was the first time I heard of this being experimental FWIW.

Reply:
Yeah, sorry that wasn't super clear

Reply:
mentioned here: https://github.com/stackabletech/issues/issues/528